src/main/java/com/thealgorithms/strings/Palindrome.java

```diff
@@ -11,27 +11,27 @@ class Palindrome {
     public static void main(String[] args) {
         String[] palindromes = { null, "", "aba", "123321" };
         for (String s : palindromes) {
-            assert isPalindrome(s) &&
+            assert isPalindromeStringBuilder(s) &&
             isPalindromeRecursion(s) &&
-            isPalindrome1(s);
+            isPalindromeTwoPointer(s);
         }
 
         String[] notPalindromes = { "abb", "abc", "abc123" };
         for (String s : notPalindromes) {
-            assert !isPalindrome(s) &&
+            assert !isPalindromeStringBuilder(s) &&
             !isPalindromeRecursion(s) &&
-            !isPalindrome1(s);
+            !isPalindromeTwoPointer(s);
```

Comment:
Put This function into the corresponding test file.

Reply:
done. Kindly review it @debasishbsws